src/testingUtils.ts

```diff
@@ -8,8 +8,7 @@ import {
 } from "./classes";
 import { ReplaySchema } from "./replaySchema";
 import * as uuidv1 from "uuid/v1";
-import { DurableEntityContext } from "./durableentitycontext";
-import { EntityContext, OrchestrationContext } from "./types";
+import { DurableEntityContext, EntityContext, OrchestrationContext } from "./types";
```

Comment:
I just realized that it's a bit confusing to have a `DurableEntityContext` and an `EntityContext` type. What is the difference between the two again?

Reply:
The `EntityContext` type is the type of the `context` argument passed to entity functions, and `DurableEntityContext` is the type of the `df` object on the `EntityContext`. If you have better suggestions for the naming, I'm open to it, but changing `DurableEntityContext` would be a breaking change. Ofc, this can be avoided if we implement #422 😛

Reply:
I see. Yeah the naming here is not ideal but I'm not ready to change this at this time :) . Let's put a pin on this, but thanks for clarifying!